ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -1243,13 +1284,18 @@ export class WalletDB {
       transaction.hash(),
       tx,
     )
+    if (this.nullifierBloomFilter) {
+      this.nullifierBloomFilter.put(nullifier)
+    }
   }
 
   async deleteNullifierToTransactionHash(
     account: Account,
     nullifier: Buffer,
     tx?: IDatabaseTransaction,
   ): Promise<void> {
+    // TODO: maybe keep a count of how many times this method was called, and
+    // reset the bloom filter after a certain threshold
```

Comment:
I don't think we need to worry about nullifiers here. Entries in `nullifierToTransactionHash` are only deleted if a pending transaction expires. The nullifier is still valid for that account, but that transaction isn't.